apps/chat2bridge/chat2bridge.nim

```diff
@@ -218,7 +237,8 @@ proc stop*(cmb: Chat2MatterBridge) {.async: (raises: [Exception]).} =
 
   await cmb.nodev2.stop()
 
-{.pop.} # @TODO confutils.nim(775, 17) Error: can raise an unlisted exception: ref IOError
+{.pop.}
+  # @TODO confutils.nim(775, 17) Error: can raise an unlisted exception: ref IOError
```

Comment:
Another comment that could be considered scrambled.

Reply:
this formatting is "correct": it would not fit on the same line, and in order to show that it "belongs" to `pop` (and not whatever comes after), it's indented.